wiki/History_of_osu!/2015/en.md

```diff
@@ -0,0 +1,33 @@
+# 2015
+
+*Main page: [History of osu!](/wiki/HOO).*
+
+## March
+
+peppy reconsiders that his current business model will not obtain him his "Private osu! G6 Jet" and enables the *[osu!coin](/wiki/osu!coin)* to hopefully increase revenue. Every current player was given 10 osu!coins to play (at the rate of 1 play = 1 osu!coin). These can be replenished after obtaining a combo of multiples of 100. Or simply by purchasing them at these rates:
+
+-   1 osu!coin = $1 USD
+-   10 osu!coins = $10 USD
+-   1000 osu!coins = $999USD (Saves you 0.001%! BEST VALUE!)
+-   Infinite osu!coins = Your first unborn child - osu! needs more team members, have your unborn child work off your debts! :)
+
+Links:
+
+- [osu!coins](https://osu.ppy.sh/home/news/2015-03-31-osucoins)
+
+## April
+
+The next day, peppy decides to rollback on the osu!coins due to popular demand. A new subdomain for the osu!web redesign was opened for preview.
+
+Links;
+
+- [peppy's remarks on osu!coins](https://disq.us/p/w2ydos)
+- [osu!next](http://osunext.tumblr.com/post/115384460628/the-super-early-ultra-alpha-buggy-read-only-of-the)
+
+## July
+
+peppy and flyte briefly explains why the temporary osu!web redesign subdomain was `jizz.ppy.sh` in an episode of osu!talk.
```

Comment:
explain

Reply:
`explained`, I was watching the video when I added that, so I mistook it as "happening now"